src/Components/CategoryCard/GenderCategoryCard.jsx

```diff
@@ -0,0 +1,19 @@
+import './category-card.css'
+
+const GenderCategoryCard = ({ source, information, caption }) => {
+    return (
+        <>
```

Comment:
Here too.

Reply:
Removed all the unnecessary react fragments